site/en/docs/semantic_retriever.ipynb

```diff
@@ -1001,17 +1001,6 @@
       "name": "semantic_retriever.ipynb",
       "toc_visible": true
     },
```

Comment:
Please revert these chunks removing the keywords. 

old versions of tensorflow_docs did that, new ones don't. To upgrade:

```
pip install -U git+https://github.com/tensorflow/docs
```